### DISCUSSION TITLE
Clarification: What ID types are supported for the "Get watched history" endpoint?

### DISCUSSION THREAD

**Discussion by henrikfroehling** [trakt/trakt-api#378] [Q&A] [answered]
The documentation for the [Sync: Get watched history](https://trakt.docs.apiary.io/#reference/sync/get-history/get-watched-history) endpoint states that an optional Trakt ID can be specified to limit the history for a specific item.

But which concrete ID types are supported?
The examples in the documentation only have Integer values. It seems, that also other ID types, like IMDB IDs, are working?

So, the following examples (Show: Breaking Bad) are also possible?
1. sync/history/shows/tt0903747 (IMDB)
2. sync/history/shows/breaking-bad (Slug)

**Comment by rectifyer** [comment, marked as answer]
Trakt ID is the only official ID type supported in the `/sync/history` method. It is possible other IDs might work, but it's undocumented and can't be relied upon.

**Reply by henrikfroehling** [reply]
Okay, thanks for clarification.
